aqandu/gaussian_model_utils.py

```diff
@@ -234,8 +281,11 @@ def setupDataMatrix(sensor_data, space_coordinates, time_coordinates, device_loc
 #    saveMatrixToFile(data_matrix, '2interpolated.txt')
     data_matrix, space_coordinates = removeBadSensors(data_matrix, space_coordinates, 0.6)
 #    saveMatrixToFile(data_matrix, '3matrix_removed_bad.txt')
-#    data_matrix, time_coordinates = trimEdgeZeroElements(data_matrix, time_coordinates)
-#    saveMatrixToFile(data_matrix, '4matrixtrimmed.txt')
+
+#  This is important because bins on either end of the time range might not have enough measurements
+#    data_matrix, time_coordinates = trimBadEdgeElements(data_matrix, time_coordinates)
+
+    #    saveMatrixToFile(data_matrix, '4matrixtrimmed.txt')
```

Comment:
Looks like there is a ton of commented out code in this section, can we remove it?

Reply:
Agreed, it should be removed.  However, I would like to come back to that as I continue to test and understand it better.  It sits in the flow right now to remind me (for the time being) of how this part used to go.

Reply:
I will put a note explaining to remove this after fully tested.

Reply:
Much of the commented code is now removed.  More to come.